src/screens/Library.jsx

```diff
@@ -0,0 +1,101 @@
+import React, {useState, useEffect} from "react";
+import likesStyles from "./Likes.module.css";
+import profileStyles from "./Profile.module.css";
+import Card from "../components/Card";
+import { API } from "../static/Services";
+import Loader from '../components/Loader';
+import buttonStyles from "../components/Button.module.css";
+
+const defaultState = {
+    labs: [],
```

Comment:
This is totally valid, and really good. However, when having only few state variables, you could declare them individually with useState.

[labs, setLabs] = useState(null)
[candidates, setCandidates] = useState(null)

This way you won't have to do all the dance of 
`setState(prevState => {
                return {
                    ...prevState,
                    labs: labs,
                    loading: ''
                }
            });`

You could just do `setLabs(data)`